.github/workflows/atomic_cache.yml

```diff
@@ -0,0 +1,96 @@
+name: ibotta/atomic_cache
+on:
+  push:
+    branches:
+    - "**/*"
+  pull_request:
+  schedule:
+  - cron: 53 9 13 * *
+concurrency:
+#   # This item has no matching transformer
+#   maximum_number_of_builds: 0
+  group: "${{ github.ref }}"
+  cancel-in-progress: true
+env:
+  RUBYGEMS_API_TOKEN: "${{ secrets.RUBYGEMS_API_TOKEN }}"
+  GITHUB_API_TOKEN: "${{ secrets.GITHUB_API_TOKEN }}"
+  encrypted_3b12ef27616a_key: "${{ secrets.ENCRYPTED_3B12EF27616A_KEY }}"
+  encrypted_3b12ef27616a_iv: "${{ secrets.ENCRYPTED_3B12EF27616A_IV }}"
+  PASSPHRASE: "${{ secrets.PASSPHRASE }}"
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+    - run: |-
+        mkdir -p ~/.ssh
+        ssh-keyscan -H github.com >> ~/.ssh/known_hosts
+    - name: checkout
+      uses: actions/checkout@v3
+    - uses: ruby/setup-ruby@v1
+      with:
+        ruby-version: "${{ matrix.rvm }}"
+    - run: gem install bundler -v 2.3.26
+    - run: bundle install --jobs=3 --retry=3
+    - run: bundle exec rake spec
+    strategy:
+      matrix:
+        rvm:
+        - 2.5
+        - 2.6
+        - 2.7
+        gemfile:
+        - Gemfile.rails52
+        - Gemfile.rails6
+        - Gemfile.rails61
+        exclude:
+        - rvm: 2.5
+          gemfile: Gemfile.rails6
+        - rvm: 2.5
+          gemfile: Gemfile.rails61
+        - rvm: 2.7
+          gemfile: Gemfile.rails52
```

Comment:
We can drop support for rails 5.2 since it's EOL https://endoflife.date/rails (either in this migration or another pr.

Reply:
Hell yeah.